sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor{TPartition}.cs

```diff
@@ -693,7 +693,7 @@ async Task performProcessing()
 
             return new PartitionProcessor
             (
-                Task.Run(performProcessing, cancellationSource.Token),
```

Comment:
Cancellation will happen within the `performProcessing` delegate; there's no benefit to having a token associated with the run request.